speedtest/edu/washington/escience/myria/mrbenchmarks/Main.java

```diff
@@ -103,10 +94,14 @@ public static void main(String[] args) throws Exception {
 
     Set<Integer> computingWorkers = new HashSet<Integer>();
     HashMap<String, Integer> workerName2ID = new HashMap<String, Integer>();
-    for (Entry<Integer, SocketInfo> e : allWorkerInfos.entrySet()) {
-      workerName2ID.put(e.getValue().getHost(), e.getKey());
+    MyriaConfiguration config =
+        MyriaConfiguration.loadWithDefaultValues(FilenameUtils.concat(masterHome, MyriaConstants.DEPLOYMENT_CONF_FILE));
+    for (String id : config.getWorkerIds()) {
+      workerName2ID.put(config.getHostname(id), Integer.parseInt(id));
     }
 
+    BufferedReader br =
+        new BufferedReader(new InputStreamReader(new FileInputStream(new File(queryWorkerNameListFile))));
```

Comment:
I simply moved this line from above to this place with no change. Yes it should be `BufferedReader(FileReader(String))`, fixed.